addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -112,10 +116,19 @@ def _search_is_self(self, operator, operand):
     def _search_is_pinned(self, operator, operand):
         if operator != 'in':
             return NotImplemented
-        return expression.OR([
-            [("unpin_dt", "=", False)],
-            [("last_interest_dt", ">=", self._field_to_sql(self._table, "unpin_dt"))],
-            [("channel_id.last_interest_dt", ">=", self._field_to_sql(self._table, "unpin_dt"))],
+        unpin_field = self._field_to_sql(self._table, "unpin_dt")
```

Comment:
This is still "hoping" that the alias is the right one, no?

How is this improving anything?

Reply:
Fixed, I clicked ready too fast last time.